recipes/scrapy/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "Scrapy" %}
+{% set version = "1.1.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "fab9c5074bc484add855d2d072ff26fdc0ba5344858cc68178f4f4f39a6f2cce" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  # Skip win+py3k as the build as the framework is not 100% functional.
+  skip: True  # [win and py3k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - scrapy = scrapy.cmdline:execute
+  number: 0
+
+# The requirements below shall match the requirements from the target package
+# version (check setup.py for changes).
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - twisted >=10.0.0
+    - w3lib >=1.15.0
+    - queuelib
+    - lxml
+    - pyopenssl
+    - cssselect >=0.9
+    - six >=1.5.2
```

Comment:
That comes straight from scrapy's `setup.py`: https://github.com/scrapy/scrapy/blob/master/setup.py#L43